src/modules/storagefacility/NodeTable.js

```diff
@@ -6,7 +6,7 @@ import MusitNode from '../../models/node';
 
 export default class NodeGrid extends Component {
   static propTypes = {
-    tableData: PropTypes.arrayOf(PropTypes.instanceOf(MusitNode)).isRequired,
+    tableData: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
maybe. It can be part of an ongoing effort for porting components to flow + better proptypes